debezium-quarkus-outbox/deployment/src/main/java/io/debezium/outbox/quarkus/deployment/OutboxProcessor.java

```diff
@@ -122,6 +127,14 @@ public void produceOutboxBuildItem(CombinedIndexBuildItem index,
         LOGGER.infof("Binding Payload as '%s'.", payloadType.name().toString());
 
         outboxEventEntityProducer.produce(new OutboxEventEntityBuildItem(aggregateIdType, payloadType));
+
+        // We enable outbox events in *all* persistence units.
```

Comment:
:+1: 

For context, I don't think it's worth trying to optimise this to enable `setXmlMappingRequired(true)` more selectively: the important matter is to not load the XML parsers at all when noone is needing them.

But if a single PU needs it, might as well keep this on for all, as classes will be loaded anyway.